eng/common/tools.sh

```diff
@@ -5,6 +5,9 @@
 # CI mode - set to true on CI server for PR validation build or official build.
 ci=${ci:-false}
 
+# Docker mode - set to true on docker builds.
```

Comment:
Unless we know an identical change is coming in the next Arcade update, we can't change eng/common/ files. They'll just get reverted on the following Monday.

Suggest instead adding a build step to ci.yml in the job that keeps failing:
``` yaml
      - task: NuGetCommand@2
        displayName: 'Workaround NuGet issues'
        condition: succeeded()
        inputs:
          command: custom
          arguments: '--version'
```

If it has to be `dotnet nuget --version` (@kartheekp-ms, @rainersigwald❔), can use the [`DotNetCoreCLI@2` task](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/build/dotnet-core-cli?view=azure-devops). Either include the [`UseDotNet@2` task](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/dotnet-core-tool-installer?view=azure-devops) first (with the `useGlobalJson` task input) or execute the previous task outside the sources directory.

Reply:
Thanks for letting me know.